schema.sql

```diff
@@ -58,3 +58,13 @@ CREATE TABLE visits (
   date_of_visit DATE,
   PRIMARY KEY(id)
 );
+
+/* Perfomance audit */
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+CREATE INDEX animal_id_asc ON visits (animal_id ASC);
+
+CREATE INDEX vet_id_asc ON visits (vet_id ASC);
+
+CREATE INDEX email_asc ON owners (email ASC);
+
```

Comment:
## Optional

- [ ] I suggest making the index on the case insensitive version of the email, 
As email addresses are not case sensitive, we might need to look for the emails in a case insensitive way. There is an option to reflect that in the index by using:
`CREATE INDEX ON owners ((lower(email)));`

Reply:
Thank you